spec/spec_helper.rb

```diff
@@ -1 +1,23 @@
+if ENV['COVERAGE']
```

Comment:
if its not just for simpleconv then why is it wrapped in a `if ENV['COVERAGE']` ?